### ISSUE TITLE
Demo Site missing code

### ISSUE THREAD

**Issue by v-rusmit** [BlazorFluentUI/BlazorFluentUI#382]
When you click show code on  https://www.blazorfluentui.net nothing shows on any item

![image](https://user-images.githubusercontent.com/13225770/131234708-8a299200-6afe-40ae-b09a-4b5965ff975b.png)

**Comment by vnbaaij** [comment]
@limefrogyank There is anerror in the generator that is not occruing lacally:
`blazor.webassembly.js:1 
        
       crit: Microsoft.AspNetCore.Components.WebAssembly.Rendering.WebAssemblyRenderer[100]
      Unhandled exception rendering component: Object reference not set to an instance of an object.
System.NullReferenceException: Object reference not set to an instance of an object.
   at BlazorFluentUI.Demo.Shared.Components.Demo.OnInitializedAsync()
   at Microsoft.AspNetCore.Components.ComponentBase.RunInitAndSetParametersAsync()`
You haveany ideawhat's causing this?

**Comment by limefrogyank** [comment]
I'm not sure. I'm away from my main pc until Friday, so I won't be able to test until then. I remember having trouble getting the source generator to work with the github action, but I'm not sure what I did to fix it.

**Comment by ahussain-vasont** [comment]
I would really like this to work the sooner possible, having a hard time getting some of the controls to work without a sample code. Please do it the sooner you can, it'd really help. Thanks for providing this.

**Comment by limefrogyank** [comment]
In the meantime, all of the samples code from the demo is on the github site. It should be in the shared folder.

Lee McPherson
________________________________

**Comment by vnbaaij** [comment]
@ahussain-vasont you can always fall back to the pages in the Shared project. That is what is being used to show the samples

**Comment by zoran123456** [comment]
So 30 days have passed and still no fix? :) I understand this is an open-source project but come on! And yes, I am perfectly aware someone else (including me) can start to make a fix on it, but honestly taking another project and looking at the code seems incredibly slow and intimidating process, especially when the original website author could probably fix it in 10 minutes.

**Comment by vnbaaij** [comment]
Correct, no fix found. As you can see from the posts above, attention has been given to this but unfortunately no solution has been found. What makes it harder is that it is working locally. Assuming it is a 10 minute fix is incredibly condescending. 
As explained above you can use the pages in the Shared project as a reference.

**Comment by zoran123456** [comment]
It is possible to use examples through 

https://blazorfluentui.azurewebsites.net/


It is not the latest version but everything seems to be working fine.

Regards

**Comment by limefrogyank** [comment]
It’s an issue with GitHub actions and how the site is compiled remotely.  When you compile locally it works.  We do that with the serverside version.  The clientside (wasm) version works locally, but when compiled remotely, it breaks the code generation project.  It’s harder to fix than you’d think 😉

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Comment by vnbaaij** [comment]
Why remove? It still works as expected on the Blazor Server site and when you run it locally.

**Comment by sbacquet** [comment]
Because exposing malfunctions is not a good promotion for Blazor.
I think you should remove the feature from https://www.blazorfluentui.net/ and instead add a link to either the Server page, or the samples code.

**Comment by vnbaaij** [comment]
For now there is an alternative running at https://blazorfluentuidemoclient.azurewebsites.net/. It is running on the free tier so performance/availability might not be at it's best.
